README.rst

```diff
@@ -163,35 +163,58 @@ with:
 
 	$ grunt watch
 
-It is also possible to develop figure in docker without installing anything locally.
-The Docker image is built on top of ``openmicroscopy/omero-web-standalone:latest``, so you will have a fully functional
-omero-web environment while developing ``omero-figure``. 
-First build the Docker image:
+
+Local install
+*************
+
+You need an `omero-web` environment. You can either install it locally or use Docker (see below).
+To install locally, we recommend that you use `conda` to install `omero-py` as described 
+at https://github.com/ome/omero-py and then perform a developer install of `omero-figure`
+which will include `omero-web`:
 
 ::
 
-   $ docker build -t figure-devel .
+    $ cd omero-figure
+    $ pip install -e .
+
+See instructions at https://omero.readthedocs.io/en/latest/developers/Web/Deployment.html
+for running the development server.
 
+Using Docker
+************
+
+It is also possible to develop figure in docker without installing anything locally.
+The Docker image at ``omero-figure/Dockerfile`` is built on top of
+`openmicroscopy/omero-web-standalone:latest <https://hub.docker.com/r/openmicroscopy/omero-web-standalone>`_,
+so you will have a fully functional omero-web environment while developing ``omero-figure``. 
+First build the Docker image and specify the server you wish to connect to:
 
-To develop ``omero-figure`` you can either make all the changes within the Docker container itself by running:
 ::
 
+   $ cd omero-figure
+   $ docker build -t figure-devel .
+   $ export OMEROHOST=demo.openmicroscopy.org
+
+The preferred option is to mount the repository containing the omero-figure code. You can make the changes locally and run `grunt watch`
+as described above and see changes in the docker container. To do so, run:
+::
 
-    $ docker run -ti -e OMEROHOST=YOUR_HOST -p 4080:4080 figure-devel
+    $ docker run -ti -e OMEROHOST -p 4080:4080  -v /PATH_TO_GIT_REPO/omero-figure:/home/figure/src figure-devel
 
-The preferred option is to mount the repository containing the omero-figure code. Make the changes locally and see the changes in the docker container. To do so, run:
 
+Alternatively, to run and develop ``omero-figure`` within the Docker container itself:
 ::
-    $ docker run -ti -e OMEROHOST=YOUR_HOST -p 4080:4080  -v /PATH_TO_GIT_REPO/omero-figure:/home/figure/src figure-devel
 
+    $ docker run -ti -e OMEROHOST -p 4080:4080 figure-devel
```

Comment:
same error as in https://github.com/ome/omero-figure/pull/515/files#r1425713166

Reply:
This workflow is fine if the correct docker image is used. npm and grunt locally not needed. No need to reformulate.